src/config/home.json

```diff
@@ -1,38 +1,32 @@
 [
   {
-    "linkName": "docs/CLI/commands",
+    "linkName": "docs/Intro",
     "img": "undraw_docusaurus_mountain.svg",
     "title": "what is starkNet",
-    "text": "intrudaction to starknet: how it works, main concepts glossary, FAQs"
-  },
-  {
-    "linkName": "docs/CLI/commands",
-    "img": "undraw_docusaurus_tree.svg",
-    "title": "building on starknet",
-    "text": "useful link to help write and deploy your own contracts on starknet"
+    "text": "Introduction to StarkNet: how it works, main concepts glossary, FAQs"
```

Comment:
We don't have an intro to StarkNet yet. I would change this sentence to describe the technical docs. We don't have a glossary nor FAQs here.